src/module_2/module_2_EDA.md

```diff
@@ -0,0 +1,2197 @@
+df_users_merged2 = df_users_merged2.sort_values(by='variant_id_count', ascending=False)
+
+df_users_merged2.head()
+```
+
+
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>user_id</th>
+      <th>user_segment</th>
+      <th>user_nuts1</th>
+      <th>first_ordered_at</th>
+      <th>customer_cohort_month</th>
+      <th>count_people</th>
+      <th>count_adults</th>
+      <th>count_children</th>
+      <th>count_babies</th>
+      <th>count_pets</th>
+      <th>number_of_orders</th>
+      <th>variant_id_count</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>4033</th>
+      <td>9a4b53affbe91ca5fd0c97f6d86e8a59ae894f3c7a69a8...</td>
+      <td>Proposition</td>
+      <td>UKI</td>
+      <td>2021-11-16 09:46:26</td>
+      <td>2021-11-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>1</td>
+      <td>320</td>
+    </tr>
+    <tr>
+      <th>3796</th>
+      <td>9b5b3679033da9e1f3a4def18667e40a7763d0d3ec9d8f...</td>
+      <td>Proposition</td>
+      <td>UKK</td>
+      <td>2022-01-19 21:40:39</td>
+      <td>2022-01-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>3</td>
+      <td>110</td>
+    </tr>
+    <tr>
+      <th>3708</th>
+      <td>11c967ad95d1e3a8f9767612e2f084aad75832cf019f8a...</td>
+      <td>Proposition</td>
+      <td>UKD</td>
+      <td>2021-01-06 08:52:51</td>
+      <td>2021-01-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>5</td>
+      <td>90</td>
+    </tr>
+    <tr>
+      <th>2098</th>
+      <td>af86901cbe9be971840f20e73e8af9d733f0887163f331...</td>
+      <td>Proposition</td>
+      <td>None</td>
+      <td>2021-03-10 10:09:10</td>
+      <td>2021-03-01 00:00:00</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>3</td>
+      <td>88</td>
+    </tr>
+    <tr>
+      <th>3140</th>
+      <td>27c2c38327ba83867c6291ee6bd0c35548edd256ed51c9...</td>
+      <td>Proposition</td>
+      <td>UKL</td>
+      <td>2020-07-13 20:25:43</td>
+      <td>2020-07-01 00:00:00</td>
+      <td>2.0</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>9</td>
+      <td>85</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+
+```python
+sns.heatmap(df_users_merged2[['number_of_orders', 'variant_id_count']].corr(), annot=True, cmap='coolwarm')
+plt.show()
+```
+
+
+    
+![png](module_2_EDA_files/module_2_EDA_36_0.png)
```

Comment:
What are you trying to see here? that users with more orders have more regulars?